woocommerce/wc-admin
# Sentence case all the things analytics

Fixes #6186

This PR updates all labels and strings in the analytics section of WooCommerce Admin to use sentence case. This is part of a broader effort accross WooCommerce / WordPress to unify around this editorial style. 

For reference: p9Jlb4-1ly-p2#comment-1999

### Detailed test instructions:

-   run this branch
-  Check each page of the analytics, and all available sub-menus to ensure all strings are now presented in sentence case
